benchmarks/benchmark_template.py

```diff
@@ -0,0 +1,18 @@
+"""
+Basic TARDIS Benchmark.
+"""
+from asv_runner.benchmarks.mark import skip_benchmark
+from benchmarks.benchmark_base import BenchmarkBase
+
+
+@skip_benchmark
+class BenchmarkXx(BenchmarkBase):
```

Comment:
Can you do that?

Reply:
Question, do you mean that add-in the official documentation in TARDIS?

If the answer is yes, I recommend a new page below the unit test section: https://tardis-sn.github.io/tardis/contributing/development/running_tests.html.

Otherwise, please let me know what is the best place to document it.

Reply:
Yes, and that location is appropriate

Reply:
Done in the commit: dca5a85f5a55e822301c61319dbcc5e7caa96f24.

![Screenshot to the new documentation for benchmark](https://github.com/tardis-sn/tardis/assets/831380/5ab875fc-dd03-4580-abd9-d22fbf3acda5)